frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -662,8 +662,8 @@ To:
         text = _("File search"),
         help_text = _([[Search a book by filename in the current or home folder and its subfolders.
 
-A search for '*' will show all files.
-Search string supports Lua patterns.
+Wildcards for one '?' or more '*' letters can be used.
+A search for '*' will show all files, 'book*' will show just files beginning with 'book' whereas '*.epub' will show all files ending in '.epub'.
```

Comment:
`'book*' will show just files beginning with 'book` Isn't this wrong ? We don't anchor at the beginning, so "book*" will match "The KOreader Book.epub", no ?

Reply:
Yes, my mistake.

Reply:
So we have a mix of wildcards and Lua patterns.
Standard "ls book*" shall not find "abook".

Reply:
> So we have a mix of wildcards and Lua patterns.

Where ?

> Standard "ls book*" shall not find "abook".

But unix shell and ls somehow anchor, and indeed requires `*book*` to find "book" at any place.
But our users would expect a search for "James" to matches "The adventures of James Bond". We can't expect them to need to use `*James*`...
But then, indeed, we don't have a way to find all books exclusively starting with "James" - but I think that's ok (we couldn't previously).

Reply:
Yes and no. I just replace "*" with ".*" and "?" with "." and "." with "%." All the other Lua patterns remain unchanged. So if someone wants names starting with "book" he just has to enter "^book*", (similar for ending in ".txt" enter "*.txt$").

Yes it is a mixture of Lua patterns and globbing.

Reply:
> But our users would expect a search for "James" to matches "The adventures of James Bond".

I am not saying it is no good, just that it's not a pure wildcards in the meaning of Unix/DOS/Windows.